extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -454,7 +454,6 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     }
     const loaderContext = new GmailLoaderContext(this.factory, msgEl, attachmentsContainerInner);
     attachmentsContainerInner = $(attachmentsContainerInner);
-    attachmentsContainerInner.parent().find(this.sel.numberOfAttachments).hide();
```

Comment:
I had to remove this line for the Gmail notes about scanned attachments to show up even with FlowCrypt enabled:

<img width="428" alt="image" src="https://github.com/FlowCrypt/flowcrypt-browser/assets/46025304/032b8566-dd33-4e1d-9b66-d88668fafcd5">